From: James Page <james.page@ubuntu.com>
Date: Mon, 12 Aug 2013 12:33:04 +0100
Subject: Disable animal-sniffer compatibility checking

Limited value in distro package build and causing compilation failures.
---
 pom.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pom.xml b/pom.xml
index 8eaecdd..2ba4629 100644
--- a/pom.xml
+++ b/pom.xml
@@ -550,7 +550,7 @@ THE SOFTWARE.
           <prepareVerifyArgs>-P release,sign</prepareVerifyArgs>
           <!-- work around for a bug in javadoc plugin that causes the release to fail. see MRELEASE-271 -->
           <preparationGoals>clean install</preparationGoals>
-          <goals>-DskipTests -Danimal.sniffer.skip=false javadoc:javadoc deploy javadoc:aggregate</goals>
+          <goals>-DskipTests -Danimal.sniffer.skip=true javadoc:javadoc deploy javadoc:aggregate</goals>
           <pushChanges>false</pushChanges>
           <localCheckout>true</localCheckout>
           <tagNameFormat>jenkins-@{project.version}</tagNameFormat>
